package-lock.json

```diff
@@ -4374,9 +4374,9 @@
       }
     },
     "@mate-academy/scripts": {
-      "version": "1.2.3",
-      "resolved": "https://registry.npmjs.org/@mate-academy/scripts/-/scripts-1.2.3.tgz",
-      "integrity": "sha512-clsxVN4sQap5pgFev+WOEi9GVSYU5/C6puBgwGFmwePMTK+ssFFw+tdE5agrKkqmRsEgT38+wRbzITh+rZs1Yw==",
+      "version": "1.2.8",
+      "resolved": "https://registry.npmjs.org/@mate-academy/scripts/-/scripts-1.2.8.tgz",
+      "integrity": "sha512-MqvuqrG8UUzQkRc375ZUIOd23nJ0BYqae/Nn5t01aDutSqZnz1ye65W4sLHiSuQJGIuHRO0CEyJxAO72wX1efw==",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.